app/src/main/java/com/example/wearit/data/AppViewModel.kt

```diff
@@ -159,11 +161,14 @@ class AppViewModel(application: Application) : AndroidViewModel(application) {
     fun saveOutfit() {
         viewModelScope.launch(Dispatchers.IO) {
 
+            if (!getAllOutfits.value.any { it.itemsInOutfit == _uiState.value.currentSelection }
```

Comment:
Dodaj sorted() do itemsInOutfit i currentSelection. Czyli it.itemsInOutfit.sorted() == _uiState.value.currentSelection.sorted()

Bo bez tego wciąż będzie można zapisać takie same outfity jeśli itemki będą ułożone w innej kolejności. Co prawda taka sytuacja, gdzie itemki będą ułożone w różnej kolejności nie powinna mieć miejsca jesli poprawisz komentarz na górze, ale zróbmy to już porządnie